cocos/core/animation/marionette/graph-eval.ts

```diff
@@ -1099,27 +1360,59 @@ interface TransitionMatch {
     requires: number;
 }
 
+interface InterruptingTransitionMatch extends TransitionMatch {
+    from: MotionStateEval;
+}
+
 class TransitionMatchCache {
     public transition: TransitionMatch['transition'] | null = null;
 
-    public requires = 0.0;
+    public requires = Infinity;
+
+    public hasMinimalCost (): this is TransitionMatch {
```

Comment:
I feel that this name may cause problems of understanding, "minimal" should mean the smallest of a bunch of data. But there seems to be no comparison with other data here, so it would be more accurate to use hasZeroCost？

Reply:
Good catch. Changed.